test/tests/api/redfish_1_0/event_service_tests.py

```diff
@@ -86,19 +86,28 @@ def test_get_event_subscription(self):
     def test_submit_test_event(self):
         """ Testing POST /EventService/SubmitTestEvent  """
         global task
-        server = Httpd(port=int(HTTPD_PORT), handler_class=self.EventServiceHandler)
+        # Suppose rackhd and test stack have the same localhost
```

Comment:
These changes are all about **"compatible with local rackhd"**.
Before modification, the cases in this PR, all failed in test stack "rackhd and cit/fit in the same machine."

There's no error occurs in ```tests/api-cit/redfish_1_0```, so I think that case doesn't need to be modified.